.travis.yml

```diff
@@ -27,23 +27,22 @@ jobs:
     - stage: test
       env: TOX_ENV=py34
       python: "3.4"
-      before_install: skip
```

Comment:
This shouldn't be a part of this commit.

Reply:
done